airbyte-config/persistence/src/main/resources/airbyte_configs_table.sql

```diff
@@ -0,0 +1,23 @@
+-- extensions
+ CREATE
+    EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- tables
+ CREATE
+    TABLE
+        IF NOT EXISTS AIRBYTE_CONFIGS(
+            id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
```

Comment:
I did some investigation, and the conclusion was that both big int and UUID should work. If we use UUID, we can also omit the `config_id` column.

I eventually chose big int mainly because of the argument that primary key should not be exposed externally. If we use the config UUID as the primary key and for some reason we need to update it, we end up having to update the primary key, which does not seem ideal.